src/processor/py/mod.rs

```diff
@@ -0,0 +1,1091 @@
+use crate::processor::model::Value::{BoolValue, DoubleValue, IntValue, StringValue};
+use crate::processor::model::{AnyValue, KeyValue, Resource, ScopeSpans, Span};
+use pyo3::prelude::*;
+use std::sync::{Arc, Mutex};
+
+// Wrapper for AnyValue that can be exposed to Python
+#[pyclass]
+struct PyAnyValue {
+    inner: Arc<Mutex<Option<AnyValue>>>,
+}
+
+#[pymethods]
+impl PyAnyValue {
+    #[new]
+    fn new() -> PyResult<Self> {
+        Ok(PyAnyValue {
+            inner: Arc::new(Mutex::new(Some(AnyValue {
+                value: Arc::new(Mutex::new(Some(StringValue("".to_string())))),
+            }))),
+        })
+    }
+    #[getter]
+    #[allow(deprecated)]
+    fn value<'py>(&self, py: Python<'py>) -> PyResult<PyObject> {
```

Comment:
Would this handle the scenarios when the inner value is a None? I don't know how common it is in actual usage, but I remember having to dance around it in the DD transformer.

Reply:
👍 Added None case.